components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/package.json

```diff
@@ -32,14 +32,14 @@
     "babel-preset-react": "^6.24.1",
     "copy-webpack-plugin": "^4.2.0",
     "css-loader": "^0.28.7",
+    "rimraf": "^2.6.2",
```

Comment:
What is the purpose of this change set (and below one)? 
Dependencies in the package.json should be sorted alphabetically (which is the default behavior of NPM).

Reply:
Applies for other places as well.